src/Sarif/HashData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
Not all of these `using` directives are needed. The ones that aren't will show up gray in VS; you can remove them.